# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/numpy/hotfix-distutils.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- numpy-1.26.4/setup.py.vanilla	2024-02-12 17:12:11.528000000 +0000
+++ numpy-1.26.4/setup.py	2024-02-12 17:12:25.679000000 +0000
@@ -13,6 +13,7 @@
 import builtins
 import re
 import tempfile
+import setuptools
 
 from distutils.errors import CompileError
 
